website-custom-views/src/content/api-reference/custom-view-config.mdx

```diff
@@ -174,6 +174,22 @@ If you need to apply more granular permissions for local development of a Custom
 
 For more information, see [Granular permissions for local development](/help-needed#granular-permissions-for-local-development).
 
+## `env.development.hostUriPath`
+
+The [Custom View context](/api-reference/commercetools-frontend-application-shell-connectors#custom-view-context) contains the [`hostUrl`](/api-reference/commercetools-frontend-application-shell-connectors#hosturl) property with the value of the built-in application's current URL. When working locally it is set to `http://localhost:3001/<projectKey>/<custom-view-host>` by default.
```

Comment:
@CarlosCortizasCT please check the changes I made 93c96eb60a478ad545ad6850650b801620f6d3b8 if the meaning is still intact.